[2023-08-28 20:08:54,901] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:08:54,926] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:08:54,926] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:08:54,926] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:08:54,927] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:08:54,961] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:08:55,010] {standard_task_runner.py:52} INFO - Started process 25052 to run task
[2023-08-28 20:08:55,027] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmped3s6bq3', '--error-file', '/tmp/tmp0i9j6xrf']
[2023-08-28 20:08:55,034] {standard_task_runner.py:80} INFO - Job 13: Subtask cria_pasta
[2023-08-28 20:08:55,183] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:08:55,392] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:08:55,396] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:08:55,401] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:08:55,493] {subprocess.py:85} INFO - Output:
[2023-08-28 20:08:55,545] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:08:55,652] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T230854, end_date=20230828T230855
[2023-08-28 20:08:55,734] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:08:55,845] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:18:07,042] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:18:07,083] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:18:07,088] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:18:07,089] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:18:07,089] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:18:07,143] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:18:07,206] {standard_task_runner.py:52} INFO - Started process 28739 to run task
[2023-08-28 20:18:07,227] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpxr88fafg', '--error-file', '/tmp/tmp2rc7v6lp']
[2023-08-28 20:18:07,232] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-08-28 20:18:07,443] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:18:07,642] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:18:07,646] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:18:07,649] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:18:07,756] {subprocess.py:85} INFO - Output:
[2023-08-28 20:18:07,842] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:18:08,011] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T231807, end_date=20230828T231808
[2023-08-28 20:18:08,079] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:18:08,255] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:21:47,800] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:21:47,823] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:21:47,823] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:21:47,823] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:21:47,823] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:21:47,873] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:21:47,946] {standard_task_runner.py:52} INFO - Started process 30541 to run task
[2023-08-28 20:21:47,967] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpc486r730', '--error-file', '/tmp/tmpn4wymk1o']
[2023-08-28 20:21:47,972] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-08-28 20:21:48,127] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:21:48,270] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:21:48,272] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:21:48,274] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:21:48,351] {subprocess.py:85} INFO - Output:
[2023-08-28 20:21:48,414] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:21:48,538] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T232147, end_date=20230828T232148
[2023-08-28 20:21:48,628] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:21:48,794] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:25:32,947] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:25:32,978] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:25:32,978] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:25:32,979] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:25:32,979] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:25:33,024] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:25:33,092] {standard_task_runner.py:52} INFO - Started process 32088 to run task
[2023-08-28 20:25:33,125] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmps52ocrty', '--error-file', '/tmp/tmp7ljqaept']
[2023-08-28 20:25:33,139] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-08-28 20:25:33,352] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:25:33,490] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:25:33,493] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:25:33,494] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:25:33,564] {subprocess.py:85} INFO - Output:
[2023-08-28 20:25:33,605] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:25:33,675] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T232532, end_date=20230828T232533
[2023-08-28 20:25:33,742] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:25:33,820] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:27:50,125] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:27:50,154] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:27:50,154] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:27:50,154] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:27:50,155] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:27:50,201] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:27:50,266] {standard_task_runner.py:52} INFO - Started process 348 to run task
[2023-08-28 20:27:50,301] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpkbs6h3mu', '--error-file', '/tmp/tmpfcmdpxie']
[2023-08-28 20:27:50,306] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-08-28 20:27:50,505] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:27:50,633] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:27:50,636] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:27:50,637] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:27:50,702] {subprocess.py:85} INFO - Output:
[2023-08-28 20:27:50,736] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:27:50,796] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T232750, end_date=20230828T232750
[2023-08-28 20:27:50,827] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:27:50,908] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:30:07,535] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:30:07,562] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:30:07,562] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:30:07,562] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:30:07,563] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:30:07,595] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:30:07,648] {standard_task_runner.py:52} INFO - Started process 1838 to run task
[2023-08-28 20:30:07,667] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmp4ge163y5', '--error-file', '/tmp/tmpvu_sgryi']
[2023-08-28 20:30:07,672] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-08-28 20:30:07,906] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:30:08,045] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:30:08,049] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:30:08,051] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:30:08,136] {subprocess.py:85} INFO - Output:
[2023-08-28 20:30:08,197] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:30:08,296] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T233007, end_date=20230828T233008
[2023-08-28 20:30:08,354] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:30:08,468] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:40:22,835] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:40:22,867] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [queued]>
[2023-08-28 20:40:22,868] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:40:22,868] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:40:22,868] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:40:22,920] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-07 00:00:00+00:00
[2023-08-28 20:40:23,004] {standard_task_runner.py:52} INFO - Started process 4619 to run task
[2023-08-28 20:40:23,034] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-07T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpyftbcrux', '--error-file', '/tmp/tmpcpeqm67r']
[2023-08-28 20:40:23,042] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-08-28 20:40:23,241] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-07T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:40:23,382] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-07T00:00:00+00:00
[2023-08-28 20:40:23,385] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:40:23,387] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-14"']
[2023-08-28 20:40:23,476] {subprocess.py:85} INFO - Output:
[2023-08-28 20:40:23,575] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:40:23,658] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230807T000000, start_date=20230828T234022, end_date=20230828T234023
[2023-08-28 20:40:23,696] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:40:23,786] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
